yt_dlp/extractor/graspop.py

```diff
@@ -0,0 +1,42 @@
+from .common import InfoExtractor
+from ..utils import (
+    traverse_obj,
+)
+
+
+class GraspopIE(InfoExtractor):
+    _VALID_URL = r'https?://vod\.graspop\.be/(?P<lang>fr|nl)/(?P<id>[0-9]+)/(?P<title>.*)/'
+    _TESTS = [{
+        'url': 'https://vod.graspop.be/fr/101556/thy-art-is-murder-concert/',
+        'info_dict': {
+            'id': '101556',
+            'ext': 'mp4',
+            'title': 'Thy Art Is Murder',
+            'description': 'Thy Art Is Murder @ Graspop',
+            'thumbnail': r're:https://cdn-mds\.pickx\.be/festivals/v3/global/original/.*\.jpg',
+            'formats': 'count:4',
```

Comment:
not needed. let the test script generate the test dict for you (as explained in the contributing guide's extractor template)

```suggestion
```